resource/queries.py

```diff
@@ -0,0 +1,29 @@
+
+def repos(org, after="null"):
+  return """
+{
+  organization(login: "%s") {
+    name
+    repositories(first: 100, after: %s) {
```

Comment:
Any reason not to use an f-string here?

Reply:
the { } bits in the graphql query clash with the {} syntax of fstrings

Reply:
ugh, you'd have to double up all the `{`s. how about `organization(login: "%(org)s") {` instead? so that it reads more fluently

Reply:
sure makes it easier to follow